### PR TITLE
CANParser: move can valid threshold into parser

### PR DESCRIPTION
**Pull request by sshane**
Just a refactor, does not change behavior

### CODE REVIEW

File: can/parser_pyx.pyx [hunk 1]
```diff
@@ -110,10 +106,7 @@ cdef class CANParser:
     cdef unordered_set[uint32_t] updated_addrs
 
     # Update invalid flag
-    self.can_invalid_cnt += 1
-    if self.can.can_valid:
-      self.can_invalid_cnt = 0
-    self.can_valid = self.can_invalid_cnt < CAN_INVALID_CNT
+    self.can_valid = self.can.can_valid
```
Comment (adeebshihadeh) [inline can/parser_pyx.pyx:109]: make this a property and return `self.can.can_valid`

File: can/parser.cc [hunk 2]
```diff
@@ -275,7 +275,7 @@ void CANParser::UpdateCans(uint64_t sec, const capnp::DynamicStruct::Reader& cms
 void CANParser::UpdateValid(uint64_t sec) {
   const bool show_missing = (last_sec - first_sec) > 8e9;
 
-  can_valid = true;
+  bool _valid = false;
```
Comment (adeebshihadeh) [inline can/parser.cc:278]: this is always false

**Review by adeebshihadeh — CHANGES_REQUESTED** [review]
Does this even work? Always make sure the tests are passing before requesting a review.

**Comment by sshane** [comment]
typo, my bad

File: can/parser.cc (hunk 2 above)
Comment (sshane) [reply can/parser.cc]: typo, my bad

**Review by adeebshihadeh — APPROVED** [review]

**Pull request merged**